Build and install also Cargo.

Part of Solaris distribution configuration.

--- a/config.toml.example
+++ b/config.toml.example
@@ -153,11 +153,11 @@
 # but also tools such as Cargo. This will also produce "combined installers"
 # which are used to install Rust and Cargo together. This is disabled by
 # default.
-#extended = false
+extended = true
 
 # Installs chosen set of extended tools if enables. By default builds all.
 # If chosen tool failed to build the installation fails.
-#tools = ["cargo", "rls", "rustfmt", "analysis", "src"]
+tools = ["cargo"]
 
 # Verbosity level: 0 == not verbose, 1 == verbose, 2 == very verbose
 #verbose = 0
